### PR TITLE
doc: Add The Things Uno quickstart

### PR DESCRIPTION
**Pull request by pradeeka7**
#### New documentation page (quickstart) about using The Things Uno with the Things Stack

Migrating V2 documentation to The Things Stack.
Old V2 doc:
The Things Uno: https://www.thethingsnetwork.org/docs/devices/uno/quick-start/

#### Screenshots

...

#### Changes

- ...
- ...

#### Notes for Reviewers

NOTE: I haven't migrated some sections as I don't have a Things Uno.
- @laurensslats 
- @benolayinka 

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/devices/the-things-uno/_index.md [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+Select **TheThingsNetwork** Arduino library from the search results.
+
+Select the **Install** button.
+
+When the installation finishes, close the **Library Manager** by selecting the **Close** button.
+
+{{< figure src="LibraryManager.png" alt="Library Manager" >}}
+
+## Connecting Things UNO to your Computer
+To begin, connect your Things Uno to the computer with the USB cable.
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/_index.md:31]: IDEM

File: doc/content/devices/the-things-uno/_index.md [hunk 2]
```diff
@@ -0,0 +1,432 @@
...
+Select the **Install** button.
+
+When the installation finishes, close the **Library Manager** by selecting the **Close** button.
+
+{{< figure src="LibraryManager.png" alt="Library Manager" >}}
+
+## Connecting Things UNO to your Computer
+To begin, connect your Things Uno to the computer with the USB cable. 
+
+[INSERT IMAGE]
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/_index.md:33]: These images you can just take from the TTN doc, they are not TTS specific

File: doc/content/devices/the-things-uno/_index.md [hunk 3]
```diff
@@ -0,0 +1,432 @@
...
+In the **menu bar** select **Tools &#8594; Board &#8594; Arduino Leonardo**. This will tell your Arduino IDE to upload sketches to your Things Uno board.
+
+[INSERT IMAGE]
+
+In the **menu bar** select **Tools &#8594; Port &#8594; Serial port** (the serial port associated with your Things Uno).
+
+[INSERT IMAGE]
+
+## Creating an Application
+The Things Uno (or any end device) needs to be registered with an **application** to communicate with The Things Stack. To create an application follow the steps given below:
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/_index.md:44]: IDEM

File: doc/content/devices/the-things-uno/_index.md [hunk 4]
```diff
@@ -0,0 +1,432 @@
...
+
+If you don’t have a Things Stack account, create a [free account](https://account.thethingsnetwork.org/register) before you begin. Next, sign in to [The Things Stack](https://eu1.cloud.thethings.network) with your credentials.
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/_index.md:46]:
```suggestion
If you don’t have a Things Stack account, create a [free account](https://account.thethingsnetwork.org/register) before you begin. Next, sign in to [The Things Stack](https://eu1.cloud.thethings.network) with your credentials.
```

Remove this section - don't need to direct people to create an account here

File: doc/content/devices/the-things-uno/_index.md [hunk 5]
```diff
@@ -0,0 +1,432 @@
...
+
+In the top menu, select **Applications**, or in the middle of the window, select the **Go to applications** (large) icon.
+
+{{< figure src="TheThingsStackConsole1.png" alt="The Things Stack Console" >}}
+
+On the **Applications** page, create a new application by selecting **+Add application** at the top-right of The Things Stack window.
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/_index.md:52]: Use the {{% tts %}} shortcode for The Things Stack (everywhere)

File: doc/content/devices/the-things-uno/_index.md [hunk 6]
```diff
@@ -0,0 +1,432 @@
...
+In the **menu bar** select **Tools &#8594; Port &#8594; Serial port** (the serial port associated with your Things Uno).
+
+[INSERT IMAGE]
+
+## Creating an Application
+The Things Uno (or any end device) needs to be registered with an **application** to communicate with The Things Stack. To create an application follow the steps given below:
+
+If you don’t have a Things Stack account, create a [free account](https://account.thethingsnetwork.org/register) before you begin. Next, sign in to [The Things Stack](https://eu1.cloud.thethings.network) with your credentials.
+
+In the top menu, select **Applications**, or in the middle of the window, select the **Go to applications** (large) icon.
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/_index.md:48]: Remove the instructions for adding an application. You can just link to integrations/adding-applications/ (use the ref shortcode)

File: doc/content/devices/the-things-uno/_index.md [hunk 7]
```diff
@@ -0,0 +1,432 @@
...
+Type the following lines to create two serial port objects for **Serial** and **Serial1**.
+
+```
+#define loraSerial Serial1
+#define debugSerial Serial
+```
+ 
+{{< note >}}The **Serial** object allows communication between **The Things Uno** and your **computer**. Also, the **Serial1** object allows communication between **The Things Uno** and the **Microchip LoRa module**.{{</ note >}}
+
+Type the following line to define the frequency plan of your Things Uno. Replace `REPLACE_ME` with `TTN_FP_EU868` or `TTN_FP_US915` depending on the frequency plan of your Things Uno and your country/region. (Click [here](https://www.thethingsnetwork.org/docs/devices/arduino/usage/#set-the-frequency-plan) to find the macro values for `freqPlan`).
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/_index.md:108]: Remove all links to TTN docs.

File: doc/content/devices/the-things-uno/_index.md [hunk 8]
```diff
@@ -0,0 +1,432 @@
...
+- From the LoRaWAN Device Repository
+- Manually
+
+Each registering option provides both **OTAA** and **ABP** activation methods. Let’s have a look at them one by one.
+
+### From the LoRaWAN Device Repository 
+This is the easiest way to register your Things Uno with the Things Stack.
+
+On the **Register end device** page, in the **From the LoRaWAN Device Repository** tab, under **Select the end device**, select the following mandatory fields:
+- Brand - `The Things Products`
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/_index.md:231]: Newline between paragraph and bullets

File: doc/content/devices/the-things-uno/_index.md [hunk 9]
```diff
@@ -0,0 +1,432 @@
...
+{{< figure src="RegisterEndDeviceRepo.png" alt="Register End Device from LoRaWAN repository" >}}
+
+Under the **Enter registration data**, select/fill the following mandatory fields.
+
+If you have selected `quickstart` from the **Firmware Ver.** in the previous step:
+- Frequency plan – Select `Europe 863-870 MHz (SF9 for RX2 - recommended)` for **EU_863_870** or `United States 902-928 MHz, FSB2 (used by TTN)` for **US_902_928**.
+- AppEUI – Copy the **AppEUI** from the output printed by the ***TheThingsUnoTest*** sketch.
+- DevEUI – Copy the **DevEUI** from the output printed by the ***TheThingsUnoTest*** sketch.
+- AppKey – Select **Generate** button to generate an **AppKey**.
+- End device ID – A unique human-readable identifier for your Things Uno.
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/_index.md:246]:
```suggestion
- End device ID – Give your device a unique human-readable identifier.
```

File: doc/content/devices/the-things-uno/_index.md [hunk 10]
```diff
@@ -0,0 +1,432 @@
...
+
+{{< note >}}
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/_index.md:248]: Don't copy and paste this here. Use a link, i.e {{< ref "reference/id-eui-constraints" >}}

File: doc/content/devices/the-things-uno/_index.md [hunk 11]
```diff
@@ -0,0 +1,432 @@
...
+Once registered, you will be redirected to the overview page of the newly registered device, where you can find the generated **AppKey** which we’ll need next.
+
+### Manually
+Manual registration requires providing more information than the previous option but it provides a flexible way of adding end devices. 
+
+On the **Register end device** page select the **Manually** tab.
+
+In the **Basic settings** select/fill the following:
+
+- LoRaWAN version - `MAC V1.1`
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/_index.md:281]:
```suggestion
- LoRaWAN version - `MAC V1.1`
```

This is incorrect, Uno uses MAC V1.0.2 and PHY 1.0.2 Rev B

File: doc/content/devices/the-things-uno/_index.md [hunk 12]
```diff
@@ -0,0 +1,432 @@
...
+- Frequency plan – select the frequency plan for your region. For Europe select `Europe 863-870 MHz (SF9 for RX2 – recommended)` and for the US select `United States 902-923 MHz, FB2 (used by TTN)`.
+- Select **Show advanced activation, LoRaWAN class and cluster settings** to expand the hidden section if not visible.
+- Activation mode: `Over the air activation (OTAA)` or `Activation by personalization (ABP)`(we recommend you to use `Over the air activation (OTAA)`).
+
+If you have selected `Over the air activation (OTAA)`:
+- Additional LoRaWAN class capabilities - `None (class A only)`
+- DevEUI - copy **DevEUI** from the output printed by the ***TheThingsUnoTest*** sketch.
+- JoinEUI - copy **AppEUI** from the output printed by the ***TheThingsUnoTest*** sketch.
+- AppKey - select **Generate** button.
+- NwkKey - select **Generate** button.
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/_index.md:292]:
```suggestion
- NwkKey - select **Generate** button.
```

This key doesnt exist in LoRaWAN 1.0.2

File: doc/content/devices/the-things-uno/_index.md [hunk 13]
```diff
@@ -0,0 +1,432 @@
...
+- End device ID – A unique, human-readable identifier for your Things Uno (see above for naming conversions).
+
+{{< figure src="RegisterEndDeviceManualOTAA.png" alt="Register End Device Manually OTAA" >}}
+
+If you have selected `Activation by personalization (ABP)`:
+- DevEUI - copy **DevEUI** from the output printed by the ***TheThingsUnoTest*** sketch.
+- Device address - select **Generate** button.
+- AppSKey - select **Generate** button.
+- FNwkSIntKey - select **Generate** button.
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/_index.md:301]:
```suggestion
- FNwkSIntKey - select **Generate** button.
```

These keys dont exist in 1.0.2

File: doc/content/devices/the-things-uno/troubleshooting.md [hunk 14]
```diff
@@ -0,0 +1,22 @@
+---
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/troubleshooting.md:1]: Delete this document

**Review by benolayinka — CHANGES_REQUESTED** [review]

File: doc/content/devices/the-things-uno/_index.md [hunk 15]
```diff
@@ -0,0 +1,432 @@
+---
+title: "The Things Uno"
+description: ""
+weight: 
+---
+
+{{< figure src="TheThingsUno.png" alt="CloudGate Gateway" class="float plain" >}}
+
+The [Things Uno](https://www.thethingsnetwork.org/docs/devices/uno/) is based on the [Arduino Leonardo](https://store.arduino.cc/usa/leonardo) (not the Arduino Uno) with an added **Microchip** LoRaWAN module ([RN2483](https://www.microchip.com/wwwproducts/en/RN2483) or [RN2903](https://www.microchip.com/wwwproducts/en/RN2903)). It is fully compatible with the **Arduino IDE** and existing Arduino shields.
```
[the remark this answers is not in the document]
Comment (pradeeka7) [reply doc/content/devices/the-things-uno/_index.md]: done

File: doc/content/devices/the-things-uno/_index.md [hunk 16]
```diff
@@ -0,0 +1,432 @@
...
+
+**The Things Uno** lets you easily get started with **LoRaWAN** application development using the Arduino IDE. Choose the correct Things Uno board to suit the recommended frequency plan of your country/region.
+
+## Setting up Arduino IDE
+The Arduino Integrated Development Environment (IDE) allows you to write code using the Arduino programming language (C/C++), compile (verify), and upload it to the Things Uno board. It supports Windows, Mac OS, Linux, and Chrome OS.
```
[the remark this answers is not in the document]
Comment (pradeeka7) [reply doc/content/devices/the-things-uno/_index.md:15]: done

File: doc/content/devices/the-things-uno/_index.md (hunk 2 above)
Comment (pradeeka7) [reply doc/content/devices/the-things-uno/_index.md]: no image was found for connecting the thins uno with a computer.

File: doc/content/devices/the-things-uno/_index.md (hunk 3 above)
Comment (pradeeka7) [reply doc/content/devices/the-things-uno/_index.md]: done

File: doc/content/devices/the-things-uno/_index.md (hunk 10 above)
Comment (pradeeka7) [reply doc/content/devices/the-things-uno/_index.md]: done

File: doc/content/devices/the-things-uno/_index.md (hunk 11 above)
Comment (pradeeka7) [reply doc/content/devices/the-things-uno/_index.md]: corrected

File: doc/content/devices/the-things-uno/_index.md (hunk 12 above)
Comment (pradeeka7) [reply doc/content/devices/the-things-uno/_index.md]: corrected

File: doc/content/devices/the-things-uno/_index.md (hunk 13 above)
Comment (pradeeka7) [reply doc/content/devices/the-things-uno/_index.md]: fixed

File: doc/content/devices/the-things-uno/troubleshooting.md (hunk 14 above)
Comment (pradeeka7) [reply doc/content/devices/the-things-uno/troubleshooting.md]: deleted

File: doc/content/devices/the-things-uno/_index.md (hunk 5 above)
Comment (pradeeka7) [reply doc/content/devices/the-things-uno/_index.md]: done

File: doc/content/devices/the-things-uno/_index.md (hunk 6 above)
Comment (pradeeka7) [reply doc/content/devices/the-things-uno/_index.md]: removed

File: doc/content/devices/the-things-uno/_index.md (hunk 8 above)
Comment (pradeeka7) [reply doc/content/devices/the-things-uno/_index.md:213]: done

File: doc/content/devices/the-things-uno/_index.md (hunk 9 above)
Comment (pradeeka7) [reply doc/content/devices/the-things-uno/_index.md]: fixed

File: doc/content/devices/the-things-uno/_index.md (hunk 4 above)
Comment (pradeeka7) [reply doc/content/devices/the-things-uno/_index.md]: removed

**Comment by pradeeka7** [comment]
@benolayinka In addition to your requested changes, I have done some minor edits.

File: doc/content/devices/the-things-uno/_index.md [hunk 17]
```diff
@@ -0,0 +1,412 @@
...
+Band: 868
+Data Rate: 5
+RX Delay 1: 1000
+RX Delay 2: 2000
+Total airtime: 0.00 s
+```
+
+**Copy** the Serial Monitor output to a text editor (E.g. Notepad) because you will need the **AppEUI** and **DevEUI** when you register your Things Node with {{% tts %}}.
+
+## Registering The Things UNO with {{% tts %}}
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/_index.md:188]:
```suggestion
## Registering The Things Uno with {{% tts %}}
```

File: doc/content/devices/the-things-uno/_index.md [hunk 18]
```diff
@@ -0,0 +1,412 @@
...
+- NwkSKey– Select **Generate** button.
+- End device ID – Give your device a unique human-readable [identifier]({{< ref "reference/id-eui-constraints" >}}).
+
+Select the **Register end device** button. 
+
+{{< figure src="RegisterEndDeviceRepoABP.png" alt="Register End Device from LoRaWAN repository ABP" >}}
+
+Once registered, you will be redirected to the overview page of the newly registered device, where you can find the generated **AppKey** which we’ll need next.
+
+### Manually
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/_index.md:249]:
```suggestion
### Manually
```

Let's remove the manual section. There are other instructions for manually adding devices, this isn't necessary.

File: doc/content/devices/the-things-uno/_index.md [hunk 19]
```diff
@@ -0,0 +1,412 @@
...
+- NwkSKey - select **Generate** button.
+- End device ID - Give your device a unique human-readable [identifier]({{< ref "reference/id-eui-constraints" >}}).
+
+{{< figure src="RegisterEndDeviceManualABP.png" alt="Register End Device Manually ABP" >}}
+
+Select the **Register end device** button.
+
+Your device is now registered with {{% tts %}}! You will be redirected to the overview page of the Things Uno.
+
+## Activating the Things UNO - OTAA
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/_index.md:288]:
```suggestion
## Activating the Things Uno - OTAA
```

**Review by benolayinka — COMMENTED** [review]
@nejraselimovic and Jaime do either of you have a Thing Uno you could help with screenshots with?

File: doc/content/devices/the-things-uno/_index.md (hunk 17 above)
Comment (pradeeka7) [reply doc/content/devices/the-things-uno/_index.md]: done

File: doc/content/devices/the-things-uno/_index.md [hunk 20]
```diff
@@ -0,0 +1,412 @@
+---
+title: "The Things Uno"
+description: ""
+weight: 
+---
+
+{{< figure src="TheThingsUno.png" alt="CloudGate Gateway" class="float plain" >}}
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/_index.md:7]:
```suggestion
{{< figure src="TheThingsUno.png" alt="The Things Uno" class="float plain" >}}
```

File: doc/content/devices/the-things-uno/_index.md [hunk 21]
```diff
@@ -0,0 +1,412 @@
...
+
+Select **TheThingsNetwork** Arduino library from the search results.
+
+Select the **Install** button.
+
+When the installation finishes, close the **Library Manager** by selecting the **Close** button.
+
+{{< figure src="LibraryManager.png" alt="Library Manager" >}}
+
+## Connecting Things UNO to your Computer
```
Comment (benolayinka) [inline doc/content/devices/the-things-uno/_index.md:33]:
```suggestion
## Connecting Things Uno to your Computer
```

Fix this everywhere

File: doc/content/devices/the-things-uno/_index.md (hunk 18 above)
Comment (benolayinka) [reply doc/content/devices/the-things-uno/_index.md:249]: @pradeeka7 please remove this whole "Manually" section.

**Review by benolayinka — CHANGES_REQUESTED** [review]

File: doc/content/devices/the-things-uno/_index.md (hunk 18 above)
Comment (pradeeka7) [reply doc/content/devices/the-things-uno/_index.md:249]: fixed

File: doc/content/devices/the-things-uno/_index.md (hunk 20 above)
Comment (pradeeka7) [reply doc/content/devices/the-things-uno/_index.md]: fixed

**Comment by Jaime-Trinidad** [comment]
Hi @pradeeka7  I added some TheThings Uno screenshots adding it using Device Repository on TTS, let me know if these are fine or you need other screenshots.

**Comment by pradeeka7** [comment]
> Hi @pradeeka7 I added some TheThings Uno screenshots adding it using Device Repository on TTS, let me know if these are fine or you need other screenshots.

Hi @Jaime-Trinidad,  Thanks for the images. However, the live data doesn't match with the LED example. Can you upload the sample sketch and take a screenshot?

**Comment by Jaime-Trinidad** [comment]
> > Hi @pradeeka7 I added some TheThings Uno screenshots adding it using Device Repository on TTS, let me know if these are fine or you need other screenshots.
> 
> Hi @Jaime-Trinidad, Thanks for the images. However, the live data doesn't match with the LED example. Can you upload the sample sketch and take a screenshot?

@pradeeka7 Let me know if the one I just upload is right.

**Comment by pradeeka7** [comment]
> > > Hi @pradeeka7 I added some TheThings Uno screenshots adding it using Device Repository on TTS, let me know if these are fine or you need other screenshots.
> > 
> > 
> > Hi @Jaime-Trinidad, Thanks for the images. However, the live data doesn't match with the LED example. Can you upload the sample sketch and take a screenshot?
> 
> @pradeeka7 Let me know if the one I just upload is right.

@Jaime-Trinidad - Thanks. This image is good. Can you upload a screen capture without the payload formatter (raw data)?

**Comment by Jaime-Trinidad** [comment]
> Can you upload a screen capture without the payload formatter (raw data)

@pradeeka7 from which part will be?

**Comment by pradeeka7** [comment]
@Jaime-Trinidad - I need two images for the following scenarios.

1. Capture Live data section for join-request and join-accept messages only.
image

2. Please capture live data before applying the payload formatter.
image

**Comment by Jaime-Trinidad** [comment]
> @Jaime-Trinidad - I need two images for the following scenarios.
> 
> 1. Capture Live data section for join-request and join-accept messages only.
>    image
> 2. Please capture live data before applying the payload formatter.
>    image

@pradeeka7 look at them and tell me if they are fine or you need other screenshots.

**Comment by pradeeka7** [comment]
@Jaime-Trinidad - Thanks for the images.

**Pull request merged**